toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -29,6 +31,28 @@ import (
 	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/timestamp"
 )
 
+var (
+	// Every valid line will be of the form: <package_name>.<architecture> <version>.<dist> <repo_id>
+	// For:
+	//     X.aarch64	1.1b.8_X-22~rc1.cm2		fetcher-cloned-repo
+	//
+	// We'd get:
+	//   - package_name:    X
+	//   - architecture:    aarch64
+	//   - version:         1.1b.8_X-22~rc1
+	//   - dist:            cm2
+	listedPackageRegex = regexp.MustCompile(`^\s*([[:alnum:]_.+-]+)\s+([[:alnum:]_+-]+)\s+([[:alnum:]._+~-]+)\.([[:alpha:]]+[[:digit:]]+)`)
+)
```

Comment:
The comment is incorrect, the `tdnf install` output is slightly different from `tdnf list`:
```suggestion
	// Every valid line will be of the form: <package_name> <architecture> <version>.<dist> <repo_id>
	// For:
	//     X aarch64	1.1b.8_X-22~rc1.cm2		fetcher-cloned-repo
	//
	// We'd get:
	//   - package_name:    X
	//   - architecture:    aarch64
	//   - version:         1.1b.8_X-22~rc1
	//   - dist:            cm2
	listedPackageRegex = regexp.MustCompile(`^\s*([[:alnum:]_.+-]+)\s+([[:alnum:]_+-]+)\s+([[:alnum:]._+~-]+)\.([[:alpha:]]+[[:digit:]]+)`)
)
```

Reply:
thanks, will fix